pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -166,6 +176,47 @@ func (r *rbac) setDefault() {
 	}
 }
 
+// function to set default concurrency limits for rbac reconciliation
+func (r *rbac) setDefaultConcurrency() {
```

Comment:
there is already setDefault() function to set createRbacResource to true, we should use the same function,
https://github.com/tektoncd/operator/blob/main/pkg/reconciler/openshift/tektonconfig/rbac.go#L146